<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>comedy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="input"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkboxDefault">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkboxHtml">
      <property name="text">
       <string>HTML Ansicht</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Abrufen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
